packages/elastic-sip-trunking/src/models/v1/create-access-control-list-request/create-access-control-list-request.ts

```diff
@@ -0,0 +1,10 @@
+import { IpRange } from '../ip-range';
+
+export interface CreateAccessControlListRequest {
+  /** Your name for the access control list entry. */
+  name: string;
+  /** Whether the access control list entry is enabled. You can use this to disable a list temporarily without deleting it. */
+  enabled?: boolean;
+  /** An array of all the IP ranges to create. */
+  ipRanges: Omit<IpRange, 'accessControlListId' | 'id' | 'createTime' | 'updateTime' | 'projectId'>[];
```

Comment:
Due to multiple duplication of this (`Omit` related `IpRange` type) for creation requests; could have sens to create a dedicated type for creation request only and use it.
Something like (pseudo code):  `IpRangeCreationRequest =  Omit<IpRange, 'accessControlListId' | 'id' | 'createTime' | 'updateTime' | 'projectId'>`
What do you think about ?

Reply:
This is a good idea. Fixed